components/speedreader/resources/panel/svg/contentTextOnly.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
```

Comment:
These are react components as svgs so they cannot be "assets" or a sub asset folder "icons" or "graphics"
"vectors" is vague, it can be implied as vector class interfaces.